FirebaseAuth/Sources/Utilities/FIRAuthErrorUtils.m

```diff
@@ -901,6 +919,10 @@
       return @"ERROR_REJECTED_CREDENTIAL";
     case FIRAuthErrorCodeMissingOrInvalidNonce:
       return @"ERROR_MISSING_OR_INVALID_NONCE";
+    case FIRAuthErrorCodeTenantIDMismatch:
+      return @"ERROR_TENANT_ID_MISMATCH";
+    case FIRAuthErrorCodeUnsupportedTenantOperation:
+      return @"UNSUPPORTED_TENANT_OPERATION";
```

Comment:
Should this be `@"ERROR_UNSUPPORTED_TENANT_OPERATION"`?

Reply:
yeah, good catch.